### ISSUE TITLE
I would like to be able to globally set all output into a panel

### ISSUE THREAD

**Issue by jakeparis** [markbirbeck/sublime-text-shell-command#48]
Currently, it seems this can only be done when using ShellCommand via a keyboard shortcut. I would like all output to go to a panel, including when I use the command prompt to invoke ShellCommand. Can this be done?

**Comment by markbirbeck** [comment]
I'm not quite sure what this issue is since it seems to be possible to create panels for both commands and keyboard shortcuts. Perhaps the idea is that rather than setting `"panel": true` on each command you want to be able to set it once and have it apply as the default to all of your custom commands?

Given the age of this issue, I realise it may not be relevant any more! But since I'm currently triaging all issues I'll leave it open for a bit longer, just in case it is still of interest.

**Comment by jakeparis** [comment]
Yes, you're correct. The idea is that you would set "panel: true" once and have it apply to all custom commands.
